pvlib/spa.py

```diff
@@ -1418,7 +1418,7 @@ def calculate_deltat(year, month):
                       -20+32*((y-1820)/100)**2
                       - 0.5628*(2150-y), deltat)
 
-    deltat = np.where(year > 2150,
+    deltat = np.where(year >= 2150,
```

Comment:
@cedricleroy This doesn't appear to solve an apparent discontinuity issue at year 2150. I get the following with and without this change:
```console
inputs_year, inputs_month = np.meshgrid(np.array([2149.999999999999, 2150, 2150.000000000001]), np.arange(1, 13))
calculate_deltat(inputs_year, inputs_month)
----
array([[ 328.59145556,    0.        ,  328.56800556],
       [ 328.8144    ,    0.        ,  328.74405   ],
       [ 329.03738889,    0.        ,  328.92013889],
       [ 329.26042222,    0.        ,  329.09627222],
       [ 329.4835    ,    0.        ,  329.27245   ],
       [ 329.70662222,    0.        ,  329.44867222],
       [ 329.92978889,    0.        ,  329.62493889],
       [ 330.153     ,    0.        ,  329.80125   ],
       [ 330.37625556,    0.        ,  329.97760556],
       [ 330.59955556,    0.        ,  330.15400556],
       [ 330.8229    ,    0.        ,  330.33045   ],
       [ 331.04628889,    0.        ,  330.50693889]])
```

Reply:
Possible to check again @markcampanelli?
Here is what I get with 5ee82b1:
```python
from pvlib.spa import calculate_deltat
>>> calculate_deltat(2150, 6)
0.0
```
With 9aee4fc:
```python
from pvlib.spa import calculate_deltat
>>> calculate_deltat(2150, 6)
329.4486722222225
```

Reply:
Sorry if I messed this up. I will check again and report back.

Reply:
Indeed the conda environment I was using wasn't grabbing the checkout, and I screwed up the year between checkouts in my first check which made me think I actually was using different code in Jupyter. Everything looks good:
```python
inputs_year, inputs_month = np.meshgrid(np.array([2149.999999999999, 2150, 2150.000000000001]), np.arange(1, 13))
calculate_deltat(inputs_year, inputs_month)
```
```python
array([[328.59145556, 328.56800556, 328.56800556],
       [328.8144    , 328.74405   , 328.74405   ],
       [329.03738889, 328.92013889, 328.92013889],
       [329.26042222, 329.09627222, 329.09627222],
       [329.4835    , 329.27245   , 329.27245   ],
       [329.70662222, 329.44867222, 329.44867222],
       [329.92978889, 329.62493889, 329.62493889],
       [330.153     , 329.80125   , 329.80125   ],
       [330.37625556, 329.97760556, 329.97760556],
       [330.59955556, 330.15400556, 330.15400556],
       [330.8229    , 330.33045   , 330.33045   ],
       [331.04628889, 330.50693889, 330.50693889]])
```